types/webpack/index.d.ts

```diff
@@ -919,7 +919,7 @@ declare namespace webpack {
     }
 
     class WatchIgnorePlugin extends Plugin {
-        constructor(paths: RegExp[]);
+        constructor(paths: string[] | RegExp[]);
```

Comment:
Looks like `(string | RegExp)[]` (or `Array<string | RegExp>`) might be more accurate.